bokeh/command/subcommand.py

```diff
@@ -112,10 +143,11 @@ def __init__(self, parser: ArgumentParser) -> None:
         self.parser = parser
         args = getattr(self, 'args', ())
         for arg in args:
-            flags = arg[0]
+            flags, spec = arg
             if not isinstance(flags, tuple):
                 flags = (flags,)
-            self.parser.add_argument(*flags, **arg[1])
+            spec = { field.name: value for field, value in values(spec) }
```

Comment:
This could just be `spec.__dict__`

Reply:
Oh, I guess you are also filtering, but then `values` should be `defined_values` or something that indicates that (or make a helper func that returns the dict and make things cleaner in this spot)